updateHostsFile.py

```diff
@@ -1141,11 +1200,17 @@ def get_file_by_url(url):
     url_data : str or None
         The data retrieved at that URL from the file. Returns None if the
         attempted retrieval is unsuccessful.
+
+    Note
+    ----
+    - BeautifulSoup is used in this case to avoid having to search in which
+        format we have to encode or decode data before parsing it to UTF-8.
     """
 
     try:
         f = urlopen(url)
-        return f.read().decode("UTF-8")
+        soup = BeautifulSoup(f.read(), 'lxml').get_text()
```

Comment:
Why do we need LXML explicitly?

Reply:
Because If we don't we get warnings that are impossible to take down :disappointed:

Reply:
`BeautifulSoup` can work with other parsers, though I suppose that it still complains even if you have another one installed...so okay, yeah, that's fine then.